spotify_project/tracks/factories.py

```diff
@@ -35,7 +34,5 @@ def artists(self, create, artists):
 
         self.artists.add(*artists)
 
-    album = factory.SubFactory(AlbumFactory)
```

Comment:
I'm surprised this got removed. Any reason why?

Reply:
I believe it was the cause of my problem. I was getting an error, because some `Track` in the db referenced an `Album` that wasn't in the db. I didn't find much on exactly how `SubFactory()` works, but when I switched to just setting `Track.album` manually, it worked.

Reply:
That is surprising to me. Were you building a factory without creating it? Something like `TrackFactory.bulid(...)`? That might have caused the issue. Hard to say. I wouldn't mind seeing the stack trace + code that was causing this issue, but we can deal with that later.

Reply:
I am using `TrackFactory.build(...)`. It probably has to do with the ugly thing I did with `BaseTestCase.add_dummy_objects_to_db`.

I know this line that I removed should be correct, because it's what all the documentation does.

Reply:
`TrackFactory.build(...)` is likely why you had to drop the SubFactory. That would be my guess. It has to do with FK shenangians.